opentelemetry-sdk/src/opentelemetry/sdk/trace/export/__init__.py

```diff
@@ -77,8 +77,8 @@ def on_end(self, span: Span) -> None:
             try:
                 self.span_exporter.export((span,))
             # pylint: disable=broad-except
-            except Exception as exc:
-                logger.warning("Exception while exporting data: %s", exc)
+            except Exception:
+                logger.exception("Exception while exporting Span.")
```

Comment:
Right, I should have mentioned that in the description but I think that warning was actually wrong here since an exception bubbling up till here is IMHO a bug in the exporter (or at least it usually means that the spans that were to be exported are permanently lost).